data/faq/is_mentor_necessary.md

```diff
@@ -0,0 +1,5 @@
+No! You just have to choose a project and write an awesome proposal. Mentors are set after we choose the projects (which is done by choosing the students).
+
+Our policy is that we'll always mentor awesome projects, so as long as your proposal is one of the wining ones, there will be a mentor for the project. But as for all other projects it might help to talk to your possible mentors, and in that case it also means trying to maybe find a mentor or at least talk to devs/maintainers about your ideas.
+
+Note however, that it is always a good idea to discuss your proposal and project milestones' feasibility with the developers and the larger coala community on [gitter](https://coala.io/chat). You will almost always get valuable feedback for your project which in turn will increase your chances of getting your GSoC proposal accepted!
```

Comment:
Please add linebreaks :D linelength 80 and it is a nice thing to start every sentence in a new line. This makes changing text in the future much easier.
so instead of the first paragraph you could do this:"
No!
You just ...
bla bla proposal.
Mentors are set....